specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2018-07-01/storagesync.json

```diff
@@ -3190,6 +3547,12 @@
           "maximum": 100,
           "description": "Level of free space to be maintained by Cloud Tiering if it is enabled."
         },
+        "tierFilesOlderThanDays": {
+          "type": "integer",
```

Comment:
This type is used several times. I would recommend to declare this type in `definitions` and then use `$ref`.

Reply:
@sergey-shandar Will WontFix work on this comment ? consistent with other properties defined ?